### PR TITLE
Multicast messages

### PR DESCRIPTION
**Pull request by gb2233**
Enables groups of servers to share local chats
`lists=[
                [ 'skyblock1' , 'skyblock2' ], //server group 1
                [ 'minigameminigame1' , 'minigame2' ], //server group 1
              ]`

### CODE REVIEW

File: src/main/java/dev/aura/bungeechat/config/Configuration.java [hunk 1]
```diff
@@ -352,7 +352,6 @@ private void convertYAMLConfig() {
           ImmutableMap.<String, Object>builder()
               .put("enabled", section.getBoolean("enabled"))
               .build();
-
```
Comment (BrainStone) [inline src/main/java/dev/aura/bungeechat/config/Configuration.java]: Leave that line please.

File: src/main/java/dev/aura/bungeechat/message/MessagesService.java [hunk 2]
```diff
@@ -149,6 +150,44 @@ public static void sendLocalMessage(BungeeChatContext context) throws InvalidCon
...
+    Predicate<BungeeChatAccount> isNotLocal = getLocalPredicate(localServerName).negate();
+
+    // ChatLoggingManager.logMessage(ChannelType.LOCAL, context);
+
+    sendToMatchingPlayers(finalMessage, isNotLocal, isDestination);
+
+    /*if (ModuleManager.isModuleActive(BungeecordModuleManager.SPY_MODULE)) {
+      String localSpyMessage = preProcessMessage(context, account, Format.LOCAL_SPY, false).get();
+      sendToMatchingPlayers(localSpyMessage, BungeeChatAccount::hasLocalSpyEnabled, isNotLocal);
+    }*/
```
Comment (BrainStone) [inline src/main/java/dev/aura/bungeechat/message/MessagesService.java]: Why is this commented out?

File: src/main/java/dev/aura/bungeechat/message/MessagesService.java [hunk 3]
```diff
@@ -149,6 +150,44 @@ public static void sendLocalMessage(BungeeChatContext context) throws InvalidCon
...
+      throws InvalidContextError {
+    context.require(BungeeChatContext.HAS_SENDER, BungeeChatContext.HAS_MESSAGE);
+    // Optional<BungeeChatAccount> account = context.getSender();
+    Optional<String> finalMessage = preProcessMessage(context, Format.LOCAL_CHAT);
+    String localServerName =
+        context.hasServer() ? context.getServer().get() : context.getSender().get().getServerName();
+    Predicate<BungeeChatAccount> isDestination = getServerPredicate(servers);
+    Predicate<BungeeChatAccount> isNotLocal = getLocalPredicate(localServerName).negate();
+
+    // ChatLoggingManager.logMessage(ChannelType.LOCAL, context);
```
Comment (BrainStone) [inline src/main/java/dev/aura/bungeechat/message/MessagesService.java]: Why is this commented out?

**Comment by gb2233** [comment]
I used `./gradlew spotlessApply`
The comments were accidentally left there after testing some things

**Pull request merged**